## 33% of end-end tests have passed
```
Total executed tests: 43
Failed tests: 29
Passed tests: 14
```
```
IDs of failed tests: 702845,702840,702894,703086,702947,702958,702786,702745,702731,702844,702841,703496,702839,702742,702808,702809,702838,702948,702859,702846,702869,702957,702843,702732,703503,703297,702936,703495,703194 
```
<h3>Failed tests (29)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=develop&PR_ID=16985&TR_CASE_IDS=702845,702840,702894,703086,702947,702958,702786,702745,702731,702844,702841,703496,702839,702742,702808,702809,702838,702948,702859,702846,702869,702957,702843,702732,703503,703297,702936,703495,703194">Rerun failed tests</a></li><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224428">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><p><blockquote><div>Device 2: Sending message 'hello' </div><div>Device 2: Find `ChatMessageInput` by `accessibility id`: `chat-message-input` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:1097: in test_1_1_chat_non_latin_messages_stack_update_profile_photo
    [self.chat_2.send_message(message) for message in messages]
critical/chats/test_1_1_public_chats.py:1097: in <listcomp>
    [self.chat_2.send_message(message) for message in messages]
../views/chat_view.py:989: in send_message
    self.chat_message_input.send_keys(message)
../views/base_element.py:375: in send_keys
    self.find_element().send_keys(value)
/home/jenkins/.local/lib/python3.10/site-packages/appium/webdriver/webelement.py:235: in send_keys
    self._execute(RemoteCommand.SEND_KEYS_TO_ELEMENT, {'text': ''.join(keys), 'value': keys})
/home/jenkins/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute
    return self._parent.execute(command, params)
/home/jenkins/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
/home/jenkins/.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 androidx.test.uiautomator.StaleObjectException; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
E   Stacktrace:
E   io.appium.uiautomator2.common.exceptions.StaleElementReferenceException: androidx.test.uiautomator.StaleObjectException
E   	at io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:63)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:277)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:271)
E   	at io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:435)
E   	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)
E   	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:267)
E   	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:250)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1294)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:911)
E   	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:611)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:552)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:466)
E   	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438)
E   	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:140)
E   	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)
E   	at java.lang.Thread.run(Thread.java:919)
E   Caused by: androidx.test.uiautomator.StaleObjectException
E   	at androidx.test.uiautomator.UiObject2.getAccessibilityNodeInfo(UiObject2.java:647)
E   	at androidx.test.uiautomator.UiObject2.hashCode(UiObject2.java:105)
E   	at java.lang.Object.toString(Object.java:291)
E   	at java.util.Formatter$FormatSpecifier.printString(Formatter.java:2978)
E   	at java.util.Formatter$FormatSpecifier.print(Formatter.java:2855)
E   	at java.util.Formatter.format(Formatter.java:2524)
E   	at java.util.Formatter.format(Formatter.java:2459)
E   	at java.lang.String.format(String.java:2870)
E   	at io.appium.uiautomator2.utils.ReflectionUtils.invoke(ReflectionUtils.java:85)
E   	at io.appium.uiautomator2.core.AxNodeInfoExtractor.extractAxNodeInfo(AxNodeInfoExtractor.java:58)
E   	at io.appium.uiautomator2.core.AxNodeInfoExtractor.toAxNodeInfo(AxNodeInfoExtractor.java:48)
E   	at io.appium.uiautomator2.utils.ElementHelpers.setText(ElementHelpers.java:92)
E   	at io.appium.uiautomator2.model.BaseElement.setText(BaseElement.java:156)
E   	at io.appium.uiautomator2.handler.SendKeysToElement.setText(SendKeysToElement.java:86)
E   	at io.appium.uiautomator2.handler.SendKeysToElement.safeHandle(SendKeysToElement.java:115)
E   	at io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:59)
E   	... 33 more
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b298ad72c3d4716a3b8f35f201e4c1e?auth=1c695653912430a3cc60454fad793dae#194">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/93d5ee0c472c47efbd7bac85cfe0da1c?auth=da6ee3e18fb7a7b0985e11741b9f56f5#150">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224424">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Pin feature is in development]]</b><br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224464">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><p><blockquote><div># STEP: Unmute chat</div><div>Device 1: Could not reach home view by pressing system back button </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:1399: in test_1_1_chat_mute_chat
    chat.long_press_element()
../views/base_element.py:319: in long_press_element
    element = self.find_element()
../views/home_view.py:74: in find_element
    self.wait_for_visibility_of_element(20)
../views/base_element.py:139: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 1: ChatElement by xpath:`//*[@content-desc='author-primary-name'][starts-with(@text,'receiver')]/..` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b298ad72c3d4716a3b8f35f201e4c1e?auth=1c695653912430a3cc60454fad793dae#499">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/93d5ee0c472c47efbd7bac85cfe0da1c?auth=da6ee3e18fb7a7b0985e11741b9f56f5#429">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224452">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td>

```
Test setup failed: medium/test_activity_center.py:176: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224455">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td>

```
Test setup failed: medium/test_activity_center.py:176: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224454">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td>

```
Test setup failed: medium/test_activity_center.py:176: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224451">test_navigation_jump_to</a>, id: 702936 </b></td></tr><tr><td><p><blockquote><div>Device 2: Joining community </div><div>Device 2: Find `Button` by `accessibility id`: `show-request-to-join-screen-button` </div></blockquote></p>

```
Test setup failed: medium/test_activity_center.py:176: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/5af5bea8ad1b4616a40f7dea1069e681?auth=bff347f305688ab868b4e970fdf14a86">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ef95698c76784fb5a27f7ecea28bf82d?auth=8613ad5b47638593f8b89cebdcc916f4">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224427">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><p><blockquote><div>Device 1: ChatMessageInput element not found </div><div>Device 1: Sending message 'mmmeowesage_text' </div></blockquote></p>

```
critical/test_public_chat_browsing.py:363: in test_community_copy_and_paste_message_in_chat_input
    self.channel.send_message(message)
../views/chat_view.py:988: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:121: in wait_for_element
    raise TimeoutException(
 Device `1`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cfdf9ed0b8894dc5bbb6c818fc016477?auth=4135e82e801928345eb4cfc71f82845f#103">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224444">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><p><blockquote><div>Device 1: Click until `ChatMessageInput` by `accessibility id`: `chat-message-input` will be presented </div><div>Device 1: Sending message 'some_text' </div></blockquote></p>

```
critical/test_public_chat_browsing.py:350: in test_community_navigate_to_channel_when_relaunch
    self.channel.send_message(text_message)
../views/chat_view.py:988: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:121: in wait_for_element
    raise TimeoutException(
 Device `1`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cfdf9ed0b8894dc5bbb6c818fc016477?auth=4135e82e801928345eb4cfc71f82845f#83">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224449">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><p><blockquote><div>Device 1: ChatMessageInput element not found </div><div>Device 1: Sending message 'message to delete and undo' </div></blockquote></p>

```
critical/test_public_chat_browsing.py:378: in test_community_undo_delete_message
    self.channel.send_message(message_to_delete)
../views/chat_view.py:988: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:121: in wait_for_element
    raise TimeoutException(
 Device `1`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cfdf9ed0b8894dc5bbb6c818fc016477?auth=4135e82e801928345eb4cfc71f82845f#191">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224465">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `Button` by `xpath`: `//*[@text="Contributors' test community"]` </div></blockquote></p>

```
critical/test_public_chat_browsing.py:332: in test_community_discovery
    self.home.element_by_text(self.discovery_community_attributes[0]).click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//*[@text="Contributors' test community"]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cfdf9ed0b8894dc5bbb6c818fc016477?auth=4135e82e801928345eb4cfc71f82845f#54">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224443">test_community_leave</a>, id: 702845 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224439">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224450">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224456">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224432">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception 
```

<b>[[Issue with username in PN, issue #6 in https://github.com/status-im/status-mobile/issues/15500]]</b><br/><br/></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224442">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224440">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224438">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224435">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224437">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><p><blockquote><div>Device 2: Joining community </div><div>Device 2: Find `Button` by `accessibility id`: `show-request-to-join-screen-button` </div></blockquote></p>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4678d9901c1b46ecba84c8969db41197?auth=0f22e83863a207afc03352f344d9bfd6">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/867140415fad435ba88d6f857db6cf4d?auth=d204b8beed79b81d11cb3c72362351dd">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224453">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>12. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224448">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>13. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224441">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>14. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224458">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224434">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><p><blockquote><div>Device 3: Could not reach home view by pressing system back button </div><div>Device 3: Find `ChatsTab` by `accessibility id`: `chats-stack-tab` </div></blockquote></p>

```
critical/chats/test_group_chat.py:408: in test_group_chat_offline_pn
    self.homes[i].chats_tab.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 3: ChatsTab by accessibility id: `chats-stack-tab` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6edb7d446b714596a0491bfa35df9440?auth=d345cb6c071b16159ed60968524f4944#325">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8a06faea8a14c4eaf66a2a379fe36fd?auth=5a2d4b8fecaa3e75ec1dc81701bfb6aa#190">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/64ef7d22dcf04addaa92e05f0b912798?auth=3296723e365193f085134cda385df692#201">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224425">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Pin feature is in development]]</b><br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224460">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><p><blockquote><div># STEP: Admin checks image message</div><div>Device 1: Looking for a message by text: test image </div></blockquote></p>

```
critical/chats/test_group_chat.py:357: in test_group_chat_send_image_save_and_share
    self.chats[0].chat_element_by_text(image_description).wait_for_visibility_of_element(60)
../views/base_element.py:139: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 1: ChatElementByText by xpath:`//*[starts-with(@text,'test image')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6edb7d446b714596a0491bfa35df9440?auth=d345cb6c071b16159ed60968524f4944#247">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8a06faea8a14c4eaf66a2a379fe36fd?auth=5a2d4b8fecaa3e75ec1dc81701bfb6aa#162">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/64ef7d22dcf04addaa92e05f0b912798?auth=3296723e365193f085134cda385df692#144">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224463">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><p><blockquote><div>Device 1: ChatMessageInput element not found </div><div>Device 1: Sending message 'Text message in the muted chat' </div></blockquote></p>

```
critical/chats/test_group_chat.py:549: in test_group_chat_mute_chat
    self.chats[0].send_message(muted_message)
../views/chat_view.py:988: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:121: in wait_for_element
    raise TimeoutException(
 Device `1`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element; 
 RemoteDisconnected
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6edb7d446b714596a0491bfa35df9440?auth=d345cb6c071b16159ed60968524f4944#328">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8a06faea8a14c4eaf66a2a379fe36fd?auth=5a2d4b8fecaa3e75ec1dc81701bfb6aa#209">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/64ef7d22dcf04addaa92e05f0b912798?auth=3296723e365193f085134cda385df692#220">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (14)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224447">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b298ad72c3d4716a3b8f35f201e4c1e?auth=1c695653912430a3cc60454fad793dae#309">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/93d5ee0c472c47efbd7bac85cfe0da1c?auth=da6ee3e18fb7a7b0985e11741b9f56f5#199">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224423">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b298ad72c3d4716a3b8f35f201e4c1e?auth=1c695653912430a3cc60454fad793dae#78">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/93d5ee0c472c47efbd7bac85cfe0da1c?auth=da6ee3e18fb7a7b0985e11741b9f56f5#77">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224426">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b298ad72c3d4716a3b8f35f201e4c1e?auth=1c695653912430a3cc60454fad793dae#396">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/93d5ee0c472c47efbd7bac85cfe0da1c?auth=da6ee3e18fb7a7b0985e11741b9f56f5#302">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224436">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b298ad72c3d4716a3b8f35f201e4c1e?auth=1c695653912430a3cc60454fad793dae#248">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/93d5ee0c472c47efbd7bac85cfe0da1c?auth=da6ee3e18fb7a7b0985e11741b9f56f5#168">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224430">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b298ad72c3d4716a3b8f35f201e4c1e?auth=1c695653912430a3cc60454fad793dae#458">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/93d5ee0c472c47efbd7bac85cfe0da1c?auth=da6ee3e18fb7a7b0985e11741b9f56f5#374">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224429">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b298ad72c3d4716a3b8f35f201e4c1e?auth=1c695653912430a3cc60454fad793dae#132">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/93d5ee0c472c47efbd7bac85cfe0da1c?auth=da6ee3e18fb7a7b0985e11741b9f56f5#108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224431">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b298ad72c3d4716a3b8f35f201e4c1e?auth=1c695653912430a3cc60454fad793dae#572">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/93d5ee0c472c47efbd7bac85cfe0da1c?auth=da6ee3e18fb7a7b0985e11741b9f56f5#437">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224462">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b298ad72c3d4716a3b8f35f201e4c1e?auth=1c695653912430a3cc60454fad793dae#336">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/93d5ee0c472c47efbd7bac85cfe0da1c?auth=da6ee3e18fb7a7b0985e11741b9f56f5#241">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224446">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aaf666e79a9b4b299650b8e92fea9553?auth=3bf4781cfd694b67ca8154e392a2c1f0#90">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8362b10ba74a4df8a6d60687fabf5680?auth=c7675a88fc2bac7ada439830b8c6326c#88">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224445">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aaf666e79a9b4b299650b8e92fea9553?auth=3bf4781cfd694b67ca8154e392a2c1f0#55">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8362b10ba74a4df8a6d60687fabf5680?auth=c7675a88fc2bac7ada439830b8c6326c#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224457">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cfdf9ed0b8894dc5bbb6c818fc016477?auth=4135e82e801928345eb4cfc71f82845f#368">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224461">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cfdf9ed0b8894dc5bbb6c818fc016477?auth=4135e82e801928345eb4cfc71f82845f#270">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224459">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6edb7d446b714596a0491bfa35df9440?auth=d345cb6c071b16159ed60968524f4944#145">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8a06faea8a14c4eaf66a2a379fe36fd?auth=5a2d4b8fecaa3e75ec1dc81701bfb6aa#98">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/64ef7d22dcf04addaa92e05f0b912798?auth=3296723e365193f085134cda385df692#78">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224433">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6edb7d446b714596a0491bfa35df9440?auth=d345cb6c071b16159ed60968524f4944#125">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8a06faea8a14c4eaf66a2a379fe36fd?auth=5a2d4b8fecaa3e75ec1dc81701bfb6aa#71">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/64ef7d22dcf04addaa92e05f0b912798?auth=3296723e365193f085134cda385df692#71">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.166.1",
-    "commit-sha1": "d64633e37b99cc7685a79c9475007881eda1e43b",
-    "src-sha256": "0pddzb2n1c50zw67j01npswjyrks87dvk6pyhvxd6hkawqgs8ajr"
+    "version": "v0.166.2",
+    "commit-sha1": "bd2c38db5d11e68d75440c175eec22892582e27f",
+    "src-sha256": "16kg4hwqn0cfzgsfbsx25zsmnb287lszcxrb704kh5cccflz65ra"
 }

